src/sentry/api/event_search.py

```diff
@@ -303,7 +302,7 @@ def handle_negation(negation, operator):
 
 def get_operator_value(operator):
     if isinstance(operator, Node):
-        operator = "=" if isinstance(operator.expr, Optional) else operator.text
+        operator = operator.text or "="
```

Comment:
Optional was removed in 0.10.0 in favor of a function that creates an optional Quantifier. For this reason, I had to adapt the logic to consider some text as optional if a Node has no matched text.